src/components/EncounterNotes/MatthiasNotes.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+import RoleEmbed from "../RoleEmbed/RoleEmbed";
+import GW2Skill from "../GW2Skill/GW2Skill";
+
+function MatthiasNotes() {
+  return (
+    <div>
+      <h3>Individual Jobs</h3>
+      <ul>
+        <li>
+          <RoleEmbed role="Cnd" profession="mirage" /> brings
+          <GW2Skill id="10302" />.
+        </li>
+        <li>
+          <RoleEmbed role="Boon" profession="thief" /> /{" "}
+          <RoleEmbed role="" profession="druid" /> block <GW2Skill id="34404" />
+          .
+        </li>
+        <li>
+          <RoleEmbed role="Boon" profession="thief" /> uses{" "}
```

Comment:
Update to Pwr Boon DD
```suggestion
          <RoleEmbed role="Pwr Boon" profession="daredevil" /> uses{" "}
```